testdata/src/struct/p.go

```diff
@@ -34,11 +34,10 @@ type s3 struct { // want `struct has size 24 \(size class 32\), could be 16 \(si
 }
 
 type s4 struct { // want `struct has size 40 \(size class 48\), could be 24 \(size class 32\), you'll save 33.33% if you rearrange it to:\nstruct {\n\t_  \[0\]func\(\)\n\ti1 int\n\ti2 int\n\ta3 \[3\]bool\n\tb  bool\n}`
-	b  bool
-	i1 int
-	i2 int
-	a3 [3]bool
-	_  [0]func()
+	b      bool
```

Comment:
Perhaps let’s add this sample as s9 separately so that we have a clear case of different samples.

Reply:
Fixed.